concepts/basics/.meta/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "blurb": "PowerShell is a task automation and configuration management program from Microsoft, consisting of a command-line shell and the associated scripting language.",
```

Comment:
I've never heard a shell described this way before

Reply:
Oh I lift this one straight from Wikipedia for a quick blurb 😆